packages/applications-service-api/src/utils/application.mapper.js

```diff
@@ -0,0 +1,117 @@
+const { startCase, toLower, snakeCase } = require('lodash');
+
+const NI_MAPPING = {
+	sector: [
+		{ ni: 'BC', api: 'business_and_commercial', label: 'Business and Commercial' },
+		{ ni: 'EN', api: 'energy', label: 'Energy' },
+		{ ni: 'TR', api: 'transport', label: 'Transport' },
+		{ ni: 'WA', api: 'water', label: 'Water' },
+		{ ni: 'WS', api: 'waste', label: 'Waste' },
+		{ ni: 'WW', api: 'waste_water', label: 'Waste Water' }
+	],
+	stage: [
+		{ ni: 0, api: 'draft', label: 'Draft' },
+		{ ni: 1, api: 'pre_application', label: 'Pre-application' },
+		{ ni: 2, api: 'acceptance', label: 'Acceptance' },
+		{ ni: 3, api: 'pre_examination', label: 'Pre-examination' },
+		{ ni: 4, api: 'examination', label: 'Examination' },
+		{ ni: 5, api: 'recommendation', label: 'Recommendation' },
+		{ ni: 6, api: 'decision', label: 'Decision' },
+		{ ni: 7, api: 'post_decision', label: 'Post-decision' },
+		{ ni: 8, api: 'withdrawn', label: 'Withdrawn' }
+	]
+};
+
+const mapFilterLabelToApi = (name, value) => {
```

Comment:
Do you think its worth having unit tests for the smaller functions in this file?

Reply:
I only test exported functions, and these ones are mostly there for a bit of extra readability. I will see about adding extra test cases to `buildApiFiltersFromNIApplications` and `mapApplicationFiltersToNI` though as there's only one at the moment